cfme/tests/infrastructure/test_timelines.py

```diff
@@ -113,39 +168,66 @@ def _rename_vm(self):
         new_name = self.vm.provider.mgmt.rename_vm(self.vm.name, self.vm.name + '-renamed')
         logger.info('%r new name is %r', self.vm.name, new_name)
         self.vm.name = new_name
+        logger.info('self.vm.name is now %r', self.vm.name)
+        logger.info('%r will be rebooted', self.vm.name)
         self.vm.provider.mgmt.restart_vm(self.vm.name)
         return self.vm.name
 
-    def _check_timelines(self, target):
-        """Navigate to the TL of the given target, select the category of the event and verify
-        that the tl_event of the VMEvent is present. if will return the length of the array
-        containing  the events found in that timeline.
+    def _delete_vm(self):
+        logger.info('%r will be deleted.', self.vm.name)
+        return self.vm.provider.mgmt.delete_vm(self.vm.name)
 
+    def _clone_vm(self):
+        logger.info('%r will be cloned to %r.', self.vm.name, self.vm.name + '-clone')
+        clone_name = self.vm.name + '-clone'
+        self.vm.clone_vm(vm_name=clone_name)
+        wait_for(self.vm.provider.mgmt.does_vm_exist, [clone_name], timeout='6m',
+                 message='Check clone exists failed')
 
-        Args:
-            target: A entity where a Timeline is present ( VM, Host, Cluster...)
+    def _migrate_vm(self):
+        logger.info('%r will be migrated.', self.vm.name)
+        return self.vm.migrate_vm()
+
+    def _check_timelines(self, target, policy_events):
```

Comment:
Question: Can we check timelines without policy_events?
The default value can be added ex.False

Reply:
I already do it here ``   def catch_in_timelines(self, soft_assert, targets, policy_events=False):

Reply:
I'd rename it to event_type instead and hand off some value